src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -601,14 +601,14 @@ std::string ContextAttribute::getLocation(ApiVersion apiVersion) const
 
 /* ****************************************************************************
 *
-* renderAsJsonObject - 
+* toJsonV1AsJsonObject -
 */
-std::string ContextAttribute::renderAsJsonObject
+std::string ContextAttribute::toJsonV1AsJsonObject
```

Comment:
As json is now part of the prefix, it could be dropped in the end of the name:

`toJsonV1AsObject`

Just a suggestion

Reply:
Fixed in 7e25b65c1f5bf757b5e8dda2b346dabb1a657d53